### PR TITLE
LPS-193161 | Master

### PR DESCRIPTION
**Pull request by alanafergs**
https://liferay.atlassian.net/browse/LPS-193161

### CODE REVIEW

**Comment by alanafergs** [comment]
ci:test:sf

**Comment by alanafergs** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase [hunk 1]
```diff
@@ -490,6 +490,112 @@ definition {
...
+				virtualHost = "www.able.com");
+
+			Navigator.openSpecificURL(url = "http://www.able.com:8080");
+
+			User.firstLoginPG(
+				password = "test",
+				userEmailAddress = "test@www.able.com",
+				virtualHostsURL = "http://www.able.com:8080");
+
+			Navigator.openSpecificURL(url = "http://www.able.com:8080");
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase:514]: @alanafergs do we need this even after logging in?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase [hunk 2]
```diff
@@ -490,6 +490,112 @@ definition {
...
+				portlet = "Pages");
+
+			PagesAdmin.addPage(
+				pageName = "Form Page",
+				sitePageType = "Widget Page");
+
+			PagesAdmin.configurePage(
+				columnName = "1 Column",
+				pageName = "Form Page",
+				sitePageType = "Widget Page");
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase:547]: @alanafergs probably not need this for this test specifically

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]
Nice work, @alanafergs! I just have a few questions and there is also an SF error. Thanks!

**Pull request closed without merging**